.github/fabricbot.json

```diff
@@ -114,8 +113,7 @@
         "eventNames": [
           "issue_comment"
         ]
-      },
-      "id": "O649JwbnjS"
```

Comment:
The UI/tool now removes this ID. I won't fight it as it is what we'll be using moving forward.